src/widgets/_scales.scss

```diff
@@ -4,10 +4,21 @@ scale {
     }
 
     highlight {
-        background-color: rgba(black, 0.45);
+        background-color: rgba($fg-color, 0.7);
+
+        box-shadow:
+            inset 0 0 0 1px rgba(black, 0.3),
+            inset 0 0 0 2px rgba(black, 0.03);
+
+        @if $color-scheme == "dark" {
+            background-clip: padding-box;
+            border: 1px solid transparent;
```

Comment:
By adding a border here—even transparent—it means widgets shift around when shifting between the light and dark style; see the Style Classes/Constants page in Granite Demo. We should probably always include the border to prevent affecting layout between the two styles.